src/types/Types.ts

```diff
@@ -52,9 +52,9 @@ export type Document = Node & {
  */
 export type Element = Node & {
 	localName: string;
-	namespaceURI: string;
+	namespaceURI: string | null;
```

Comment:
This seems to be part of issue #122. Do we need this in this PR?
Let's either:
- Revert this change and fix it for `Attr` and `Element` in another PR.
- Also make the fix for `Attr` in this PR. (In this case we should also notice in the comment something like "This fixes #122")

Reply:
Let's fix the other one(s) too and just fix it at the same type